<launch>
	<arg name="threshold_range" default="27.0" />
	<arg name="general_max_slope" default="4.0" />
        <arg name="clipping_height" default="0.5" />

        <node pkg="ray_ground_filter" type="ray_ground_filter" name="ray_ground_filter" args="input:=/velodyne_points" output="screen">
                <param name="threshold_range" value="$(arg threshold_range)"/>
		<param name="general_max_slope" value="$(arg general_max_slope)"/>
                <param name="clipping_height" value="$(arg clipping_height)"/>
	</node>
</launch>
